app/styles/app/modules/helpscout-link.sass

```diff
@@ -0,0 +1,21 @@
+.helpscout-link
+  position: fixed
+  bottom: 0
+  right: 2em
+  display: none
+  padding: 0.7em 1em 3.3em
+  z-index: 99
+  font-size: 16px
+  border: 2px solid $oxide-blue
+  border-radius: 2px
+  background-color: white
+  color: $oxide-blue
+  transform: translateY(2.7em)
+  will-change: transform
```

Comment:
it's pretty neat. I draws the element on a new layer so when animation happens only that element needs to be repainted